Modules/selectmodule.c

```diff
@@ -407,8 +407,16 @@ poll_register(pollObject *self, PyObject *args)
     unsigned short events = POLLIN | POLLPRI | POLLOUT;
     int err;
 
-    if (!PyArg_ParseTuple(args, "O|O&:register", &o, ushort_converter, &events))
+    if (!PyArg_ParseTuple(args, "O|O&:register",
+                          &o, ushort_converter, &events)) {
+        assert(PyErr_Occurred());
+        if (PyErr_ExceptionMatches(PyExc_OverflowError)) {
+            PyErr_SetString(PyExc_OverflowError,
+                            "eventmask value does not fit in C "
```

Comment:
all right, I will remove this change, and other similar changes where `ushort_converter` is used.